data/strings/types_strings.txt

```diff
@@ -21979,7 +21729,8 @@
     zh-Hant = 廚房用品店
 
   [type.shop.laundry]
-    en = Laundry
+    en = Laundromat
```

Comment:
It's Laundry if you turn it in and they do it.  It's Laundromat if you do it with their machines.